lab.js

```diff
@@ -0,0 +1,325 @@
+// console.log("Problem 1")
+
+// // function double(num){
+// //     return num *2
+// // }
+
+// // console.log(double(3))
+
+// const doubl = num => num *2
+// console.log(doubl(7))
+
+
+// console.log("Problem 2")
+
+// // function smallest(num1, num2){
+// //     let smaller = 0;
+// //     if (num1<num2){
+// //         smaller = num1
+// //     } else{
+// //         smaller = num2
+// //     }
+// //     return smaller
+// // }
+
+// const smallest = (num1,num2) => {
+//     if (num1<num2){return `${num1} is smaller`}
+//     else {return `${num2} is smaller`}
+// }
+
+// console.log(smallest(4,3))
+
+
+// console.log("Problem 3")
+
+// let input = [6,-1,5,4,3,2,1];
+
+// const smallestNum = arry => {
+//     let smaller = arry[0]
+//     for (let i =0; i <arry.length; i++){
+//         if (smaller < arry[i]){
+//         }    else {
+//             smaller = arry[i]
```

Comment:
if you put
if(smaller > arry[i]){
smaller = arry[i]
}
then you wouldn't need the else statement